dist/leaflet.draw-src.js

```diff
@@ -86,7 +92,7 @@ L.drawLocal = {
 				edit: 'Edit layers.',
 				editDisabled: 'No layers to edit.',
 				remove: 'Delete layers.',
-				removeDisabled: 'No layers to delete.'
+				removeDisabled: 'No layers to delete.',
```

Comment:
There seems to be a few of these, I just fixed one - I'll check the settings and fix what I find.